pkgs/by-name/pi/pidof/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchzip
+}:
+
+stdenv.mkDerivation {
+  pname = "pidof";
+  version = "unstable-2005-01-25";
+  src = fetchzip {
+    url = "http://www.nightproductions.net/downloads/pidof_source.tar.gz";
+    hash = "sha256-DsZrmBMUgnAkiteMgsYrPEJu78RueK4J8q0UX4aBNEs=";
+  };
+
+  makeFlags = [ "CC=cc" ];
+
+  # Makefile tries to change the owner :/
+  installPhase = ''
+    runHook preInstall
+    install pidof -Dt $out/bin/
+    install pidof.1 -Dt $out/share/man/man1/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A utility to display the PID number for a given process name";
+    platforms = with platforms; darwin;
+    # Manpage says "This is free software", so I guess that's what it is?
```

Comment:
No, saying that doesn't make it free.

Reply:
Technically, that little blurb *is* a license. It's an unclear and bad license but it is one.

I've written an email to the author to clarify what they mean, let's see what they say. I'm not opposed to declaring it unfree though I'd love to avoid it if possible.

Reply:
> Technically, that little blurb _is_ a license.

No, free software is not a legal term.

Reply:
Obviously not but you don't actually need to use "legal terms" to form a license. "Do whatever tf you want with it" is also a license for example and those certainly aren't "legal terms". You usually do want to use clear language to be clear but you technically don't have to.

I'll wait a few days for the author to clarify and declare it unfree afterwards.

Reply:
I agree. But my point is that the author might say they meant free as in free beer.

Reply:
Hi @Atemu, are you still working on this?

Reply:
No, feel free to take over.